oai-pmh-handler/src/main/java/no/sikt/nva/oai/pmh/handler/oaipmh/transformers/SimplifiedRecordTransformer.java

```diff
@@ -100,15 +106,40 @@ private static void appendAsSubject(String subject, OaiDcType oaiDcType) {
 
   private static void appendContributors(ResourceSearchResponse response, OaiDcType oaiDcType) {
     response.contributorsPreview().stream()
-        .map(SimplifiedRecordTransformer::toCreatorElement)
+        .map(SimplifiedRecordTransformer::toContributorElement)
```

Comment:
Changes from creator to contributor because it depends on the type of resource what role is a creator vs a contributor.